chromiumoxide_pdl/src/build/generator.rs

```diff
@@ -213,15 +213,24 @@ impl Generator {
             modules.extend(module);
         }
 
-        // fix unresolved type sizes
-        let mut refs = Vec::new();
+        // brute-force fix unresolved type sizes
+        let mut refs = VecDeque::new();
         std::mem::swap(&mut refs, &mut self.ref_sizes);
-        for (name, reff) in refs {
-            let ref_size = *self
-                .type_size
-                .get(&reff)
-                .unwrap_or_else(|| panic!(format!("No type found for ref {}", reff)));
-            self.store_size(&name, Either::Left(ref_size));
+        let mut sequential_retries = 0;
+        while let Some((name, reff)) = refs.pop_front() {
+            match self.type_size.get(&reff).copied() {
+                Some(ref_size) => {
+                    sequential_retries = 0;
+                    self.store_size(&name, Either::Left(ref_size));
+                }
+                None => {
+                    sequential_retries += 1;
+                    if sequential_retries > 10 {
```

Comment:
This makes sense to me and ensures that types declared out of order are properly resolved.